vital/types/metadata.h

```diff
@@ -334,31 +347,46 @@ class typed_metadata
 class VITAL_EXPORT metadata
 {
 public:
+// The design for this collection requires that the elements in the
+// collection are owned by the collection and are only returned by
+// value. This is why the std::unique_ptr is used. Unfortunately, not
+// all stdc implementations support maps with unique_ptrs. The
+// following is done to work around this limitation.
 #ifdef VITAL_STD_MAP_UNIQUE_PTR_ALLOWED
```

Comment:
(OT) AFAICT, this cannot ever be defined; oversight when adding it?

Reply:
At one point, there was a compiler check to see if this is supported. Sometime ago, that test disappeared for unknown reasons. It is on the list to restore the compile check and set the symbol accordingly.

Reply:
Pre-KWIVER? The symbol was added in 1d56f6cd2d6c7a5efc9e284bde55c0382edfba79, but with no compiler check. I went to go looking into why it disappeared and couldn't find where it had ever existed.

Reply:
I couldn't find the compiler check either, but I do remember seeing the code.